### ISSUE TITLE
Accessibility: Collapse icon name is confusing

### ISSUE THREAD

**Issue by Watso196** [ant-design/ant-design#49363]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/antd-reproduction-template-forked-4hmmwj?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clxafmaoi00063b6imds60i3r%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clxafmaoi00023b6ivj3u5n53%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clxafmaoi00033b6ivr3cdkfy%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clxafmaoi00053b6ishrtdh72%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clxafmaoi00023b6ivj3u5n53%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clxafmaoi00013b6iovdet92w%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Findex.js%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522id%2522%253A%2522clxafmaoi00023b6ivj3u5n53%2522%252C%2522activeTabId%2522%253A%2522clxafmaoi00013b6iovdet92w%2522%257D%252C%2522clxafmaoi00053b6ishrtdh72%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clxafmaoi00043b6ihvxaeamo%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clxafmaoi00053b6ishrtdh72%2522%252C%2522activeTabId%2522%253A%2522clxafmaoi00043b6ihvxaeamo%2522%257D%252C%2522clxafmaoi00033b6ivr3cdkfy%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clxafmaoi00033b6ivr3cdkfy%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

### Steps to reproduce

1. Turn on a screen reader of your choice
2. Use the Tab key to move keyboard focus to the button for the Collapse
3. Notice that the screen reader announces "right" as part of the button's accessible name

### What is expected?

The screen reader should announce "collapsed" or "expanded" to indicate that the control is expandable, and should not announce anything for the icon, because the "collapsed" or "expanded" announcement conveys what the icon visually represents

### What is actually happening?

Screen readers announce both "collapsed" or "expanded" and "right" for the icon, which is confusing for screen reader users

| Environment | Info |
| --- | --- |
| antd | 5.18.0 |
| React | 18.0.0 |
| System | Sonoma 14.5 |
| Browser | Chrome 125.0.6422.142 |

**Comment by xineonrie** [comment]
I think using the ">" symbol to indicate collapse is actually a well-established design convention that has been widely adopted in web design? If you got some special requirements, its might be a good idea to consider adding a separate text label to the title I think.
